ui/serializers.py

```diff
@@ -7,6 +7,28 @@
 from rest_framework.settings import api_settings
 
 from ui import models, permissions as ui_permissions
+from ui.utils import get_moira_client
+
+
+def validate_moira_lists(lists):
+    """
+    Raise a validation error if any of the moira lists in a list does not exist or is not a mailing list
+
+    Args:
+        lists(list): List of moira lists
```

Comment:
Can you change `list` to `list of MoiraList`?

Reply:
OK